cnab/.gitignore

```
@@ -0,0 +1,2 @@
+istio*
```

Since the bundle wants these files in a `charts` dir anyway, we don't need to ignore individual files if they are put into the common directory.

```suggestion
/charts
```

---

done!